scripts/buildRpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
@john-gress are you proposing that the solution should be for jenkins to have a check box for 'kibana-version'? Or potentially hard coding it in the build script in jenkins for this rpm?